website/content/en/docs/reference/metrics.md

```diff
@@ -177,3 +177,10 @@ Maximum number of concurrent reconciles per controller
 ### `controller_runtime_active_workers`
```

Comment:
This is auto-gened so you should be able to re-generate this metrics with `make docgen` after bumping the version of `kubernetes-sigs/karpenter` that this is referencing. Wanna just knock out two birds with one stone?

Reply:
I see so it docgens based on the karpenter core reference. 

Doing ` go get -u sigs.k8s.io/karpenter@release-v0.34.x` will give us the latest version of v0.34 but these changes aren't cherry picked there. 


Curious to learn how does the AWS process work. Do you break off a new branch and start developing against that version 0.35 or just continue to cherry pick into 0.34, then updating the provider?

Reply:
I bumped and did make docgen. THanks for sharing that command, seems preview already has the cluster state so happy to just bump to include update to your metrics.